sandbox/settings.py

```diff
@@ -105,6 +105,8 @@
 
 STATIC_URL = '/static/'
 
-TEMPLATE_DIRS = (os.path.join(BASE_DIR, 'accounts/templates'), )
+TEMPLATE_DIRS = (os.path.join(BASE_DIR, 'accounts/templates'),
+os.path.join(BASE_DIR, 'templates')
+)
```

Comment:
No ale mówimy tutaj przecież o "accounts/templates", a accounts to jest przecież aplikacja.